src/Apps/Artist/Routes/Overview/Components/ArtistRecommendations.tsx

```diff
@@ -14,14 +15,19 @@ export const ArtistRecommendations: React.FC<
   ArtistRecommendationsProps
 > = props => {
   const {
-    artist: { name },
+    artist: { name, related },
   } = props
 
+  const relatedArtists = related.artists.edges.map(edge => (
```

Comment:
I'm thinking about how this component handles when there are no related artists available, and realizing you'll see a `Related to ...` header (and corresponding section), but then it'll be empty.

So I think technically you'll want a fragment one level up, in the `Overview` that fetches the count of related artists.

Then, you can do `const showRecommendations = userHasLabFeature && hasRelatedArtists`, and so forth from there. Then in this component, you don't have to worry about if there are no related artists coming back, since it'll only be rendered when there are, and you won't have an 'empty' section being rendered.

Reply:
Yeah, I definitely assumed there would be at least one related artist. I'll work on this.

Reply:
I pushed some changes, including tests for the Overview component. Let me know what you think!

Reply:
Very nice! 👍